bgpd/bgp_route.c

```diff
@@ -10298,10 +10298,6 @@ static int bgp_show_neighbor_route(struct vty *vty, struct peer *peer,
 		return CMD_WARNING;
 	}
 
-	/* labeled-unicast routes live in the unicast table */
-	if (safi == SAFI_LABELED_UNICAST)
-		safi = SAFI_UNICAST;
-
```

Comment:
Is this right?  I thought the change was made to store both in same table

Reply:
look at the top of that function.

Reply:
Thanks, looking at diffs just isn't enough sometimes ;-)